src/stories/AlphabeticalBrowseBy.stories.js

```diff
@@ -0,0 +1,11 @@
+import AlphabeticalBrowseBy from "../lib-components/AlphabeticalBrowseBy"
+
+export default {
+    title: "Search / AlphabeticalBrowseBy",
+    component: AlphabeticalBrowseBy,
+}
+
+export const Default = () => ({
+    components: { AlphabeticalBrowseBy },
+    template: `<alphabetical-browse-by/>`,
+})
```

Comment:
We need another story to simulate the click action, there is storybook addon for it.